src/typescript/09-interfaces/interfaces.md

```diff
@@ -78,6 +84,12 @@ Sometimes all properties on an object don’t need to be required, so using the
 
```

Comment:
Right now I see this in the TOC:

- Solution 1
- Optional properties
- Setup 2

There should only be one exercise per objective, so I think this needs to be reworked a bit.